source/standards/logging.html.md.erb

```diff
@@ -1,108 +1,47 @@
 ---
-title: Logging
+title: Store and query logs
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Short-term querying of logs
+For incident management and debugging you should have a system in place so you can store and query your infrastructure and application logs.
 
-For the purposes of incident management and debugging, it's useful to
-have a system for storing and querying the most recent logs produced
-by your application and infrastructure.
+GDS has a shared account with [Logit](https://logit.io/), offering on-demand shared [ELK (Elasticsearch, Logstash and Kibana)](https://www.elastic.co/elk-stack) stacks as a service. ELK offers suitable short-term storage for your logs so they are both accessible and queryable.
 
-In general, tools for querying logs are not a good fit for long-term
-archival, and vice versa.
+You shouldn’t operate your own self hosted ELK stacks for logging. In the past we’ve found ELK relatively easy to set up, but long term it’s easier to use a cloud-hosted ELK solution.
 
-### Shared Elasticsearch, Logstash and Kibana (ELK) stack
+You should be aware that ELK isn’t appropriate for long term archival of logs. There are other services which are less expensive and more reliable.
 
-GDS has a shared account with [logit](https://logit.io/), which offers
-on-demand ELK stacks as a service.
+You can read the [Reliability Engineering team documentation](https://reliability-engineering.cloudapps.digital/#logging) for more information about using Logit.
```

Comment:
Can we move this to be with the bit about Logit 3 paragraphs earlier please?

Reply:
Agreed.